src/openrct2/drawing/drawing.h

```diff
@@ -363,6 +363,8 @@ void shorten_path(utf8 *buffer, size_t bufferSize, const utf8 *path, sint32 avai
 void scrolling_text_initialise_bitmaps();
 sint32 scrolling_text_setup(rct_string_id stringId, uint16 scroll, uint16 scrollingMode);
 
+void FASTCALL gfx_get_sprite_size(sint32 image_id, sint32* width, sint32* height);
```

Comment:
This is to retrieve the size of the OpenRCT2 logo. I felt (and still feel) there's alot of potential to use a function like this in various places throughout the code.

```
    gfx_get_sprite_size(SPR_G2_LOGO, &logoWidth, &logoHeight);
    gfx_draw_sprite(dpi, SPR_G2_LOGO, x - (logoWidth / 2), y, 0);
    y += logoHeight + 20;
```
https://github.com/wolfreak99/OpenRCT2/blob/3e132540e8b9cc90a3365bca63b2b1d7d64a0c5a/src/openrct2/windows/about.c#L221-L223

(Though I should switch it from sint32 to sint16 to be more accurate, since that's what `rct_g1_element->width` and `rct_g1_element->height` are returning.)

Your opinions?